openapi.yaml

```diff
@@ -301,61 +273,79 @@ paths:
     get:
       operationId: getRasSearchRuns
       summary: Get Runs from Query
+      description: |
+        API endpoint to query the Result Archive Store (RAS) for a list of runs based on the given search criteria.
+
+        If no search criteria is provided then the first page of resutls will be returned. 
+
+        If no timeframe is specified in the query then the results will update as new tests are run which may result in duplicate and/or missing test runs.
+        
       tags: 
       - Result Archive Store API
       parameters:
         - name: sort
           in: query
-          description: provides sorting, to:asc or to:desc
+          description: Sorts the returned runs based on the sort field, to:asc or to:desc
+          required: true
           schema: 
             type: string
+          example: requestor:asc
         - name: result
           in: query
-          description: Result Name
+          description: Result Status for the run
           schema:
             type: string
+          example: Passed
         - name: bundle
           in: query
-          description: Bundle Name
+          description: The name of the Bundle bundle that this test run is part of 
           schema:
             type: string
+          example: dev.galasa.inttests
         - name: requestor
           in: query
-          description: Requestor Name
+          description: Name of the test Requestor / submitter
           schema:
             type: string
+          example: galasa  
         - name: from
           in: query
-          description: Date Number From
+          description: Retrieve runs from the given date and time
           schema:
             type: string
             format: date-time
+          example: '2023-04-11T09:42:06.589180Z' 
         - name: to
           in: query
-          description: Date Number To
+          description: Retrieve runs up to the given date and time
           schema:
             type: string
             format: date-time
+          example: '2023-04-11T09:43:27.324075Z"'
         - name: testname
           in: query
-          description: Test Class Name
+          description: The full test name (package + short test name)
           schema:
             type: string
+          example: dev.galasa.inttests.simbank.local.mvp.SimBankLocalJava11UbuntuMvp   
         - name: page
           in: query
-          description: Page Number
+          description: The desired output page number
           schema:
             type: number
+          example: 2  
         - name: size
           in: query
-          description: Request Size
+          description: The number of test results outputted per page 
           schema:
             type: number
+          example: 20
         - name: runId
```

Comment:
Are these flags case sensitive ? This seems to set a precedence for mixed camel-case query parameters ? In which case runName might be easier to read instead of the 'runname' we have so far ?